experimental/web/sample_static/device_sync.c

```diff
@@ -13,13 +13,10 @@ iree_status_t create_device_with_static_loader(iree_allocator_t host_allocator,
   iree_hal_sync_device_params_t params;
   iree_hal_sync_device_params_initialize(&params);
 
-  // Load the statically embedded library.
-  const iree_hal_executable_library_header_t** static_library =
-      mnist_linked_llvm_library_query(
-          IREE_HAL_EXECUTABLE_LIBRARY_LATEST_VERSION,
-          /*reserved=*/NULL);
-  const iree_hal_executable_library_header_t** libraries[1] = {static_library};
-
+  // Register the statically linked executable library.
+  const iree_hal_executable_library_query_fn_t* libraries[] = {
+      mnist_linked_llvm_library_query,
+  };
```

Comment:
fyi I can add these experimental/web samples to some CI pipeline soon

maybe around here: https://github.com/google/iree/blob/572fcef510c17b70ad3e40078e597428f1b7639e/build_tools/cmake/build_runtime_emscripten.sh#L50-L51